src/services/createCampaignForGivethioProjects/createCampaignForGivethioProjects.service.js

```diff
@@ -0,0 +1,99 @@
+const errors = require('@feathersjs/errors');
+const config = require('config');
+const logger = require('winston');
+const { findCampaignByGivethIoProjectId } = require('../../repositories/campaignRepository');
+const { getGivethIoAdapter } = require('../../adapters/adapterFactory');
+
+const givethIoAdapter = getGivethIoAdapter();
+module.exports = function aggregateDonations() {
+  const app = this;
+
+  const service = {
+    async create(data, params) {
+      const { txHash, image, slug } = data;
+
+      const projectInfo = await givethIoAdapter.getProjectInfoBySLug(slug);
+      const {
+        id: givethIoProjectId,
+        title,
+        description,
+        walletAddress: ownerAddress,
+      } = projectInfo;
+      if (params.user.address !== ownerAddress) {
+        throw new errors.Forbidden('The owner of project in givethIo is not you');
+      }
+      let campaign = await findCampaignByGivethIoProjectId(app, givethIoProjectId);
+      if (campaign) {
+        throw new errors.BadRequest('Campaign with this givethIo projectId exists');
+      }
+      campaign = await app.service('campaigns').create({
```

Comment:
Will campaign service allow creating a campaign with this ownerAddress? (ownerAddress doesn't have project owner role)